CrisisTextLine/Elmer
# Mike/melted dump to pooh/at 387

#### What's this PR do?

- moves the `meltedDataDumpToCSV.js` job from `jobs/training` to `tasks` to dump CSV to Pooh
- pulls data from `autoForceMelt.js` into a json file that is converted into a CSV to dump into Pooh

#### Where should the reviewer start?
#### How should this be manually tested?
comment out everything in `autoForceMelt.js` that actually force melts users. It should produce `poohMeltedData.json`. Then run `forceMeltDataToPooh.js`, which should produce a CSV from that data.

For the `meltedDataDumpToCSV.js`, run it and it should produce a CSV from the data. 
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Questions:
